include/wx/graphics.h

```diff
@@ -132,6 +132,49 @@ class WXDLLIMPEXP_CORE wxGraphicsObject : public wxObject
     wxDECLARE_DYNAMIC_CLASS(wxGraphicsObject);
 };
 
+// ----------------------------------------------------------------------------
+// wxGraphicsPenInfo describes a wxGraphicsPen
+// ----------------------------------------------------------------------------
+
+class wxGraphicsPenInfo : public wxPenInfo
+{
+public:
+    wxGraphicsPenInfo()
+    : wxPenInfo()
+    {
+        m_widthF = -1.0;
+    }
+
+    explicit wxGraphicsPenInfo(const wxColour& colour, double widthF = 1.0, wxPenStyle style = wxPENSTYLE_SOLID)
+    : wxPenInfo(colour, 0, style)
+    {
+        m_widthF = widthF;
+    }
+
+    // Setters for the various attributes. All of them return the object itself
+    // so that the calls to them could be chained.
+
+    wxGraphicsPenInfo& Colour(const wxColour& colour);
+
+    wxGraphicsPenInfo& Width(int width);
+
+    wxGraphicsPenInfo& Style(wxPenStyle style);
+    wxGraphicsPenInfo& Stipple(const wxBitmap& stipple);
+    wxGraphicsPenInfo& Dashes(int nb_dashes, const wxDash *dash);
+    wxGraphicsPenInfo& Join(wxPenJoin join);
+    wxGraphicsPenInfo& Cap(wxPenCap cap);
+
+    wxGraphicsPenInfo& WidthF(wxDouble widthF)
```

Comment:
I'm not sure if we use `FooF` naming convention anywhere else and it seems not very clear to me, especially for a function taking `double`.

In `wx/gdicmn.h` we have `wxRealPoint` etc as counterparts to `wxPoint`, so I think it would be more consistent to use `RealWidth()` here, although, admittedly, this does look a bit confusing (is the other width unreal then?). Maybe `WidthReal()` would be more clear, even though it sounds undoubtedly worse.